lib/prototyp/bigint.go

```diff
@@ -30,11 +43,16 @@ func NewBigIntFromHexString(hs string) BigInt {
 
 func NewBigIntFromString(s string, base int) BigInt {
 	b := big.NewInt(0)
+	if base != 0 && (2 > base || base > big.MaxBase) {
```

Comment:
this condition doesn't read very well

Reply:
it is horrible, but it the same conditon which is inside `math` package and which leads to panic. I can add a comment with link to that code.